src/models/data.py

```diff
@@ -323,6 +330,18 @@ def __iter__(self):
     def __len__(self) -> int:
         return len(self.data_files) // self.batch_file_size
 
+    @staticmethod
+    def prepare_mask(spatial_mask: Optional[xr.DataArray]) -> Optional[xr.DataArray]:
```

Comment:
do we definitely want the dataloader to be inverting the mask and not just use it 

`xr.Dataset.where(~mask)` ?

Leave this up to you i actually don't have a preference just interested in the reasoning for this inversion here!

Reply:
`where` is so much more elegant. I'll make that change.

In terms of the inversion, I was trying to think about which would be more intuitive - I figured since we call it a `mask`, then probably where `mask == 1` would be "masked out" (i.e. removed).

Reply:
Updated!